api/openapi_server/models/schema.py

```diff
@@ -1,12 +1,209 @@
+        load_instance = True
+
+
+class ImageTagTypeSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = ImageTagType
+        include_relationships = True
+        load_instance = True
+
+
+class MatchResultSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = MatchResult
+        include_relationships = True
+        load_instance = True
+
+
+class GroupMatchResultSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = GroupMatchResult
+        include_relationships = True
+        load_instance = True
+
+
+class MatchFailureSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = MatchFailure
+        include_relationships = True
+        load_instance = True
+
+
+class HousingProgramServiceProviderSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = HousingProgramServiceProvider
+        include_relationships = True
+        load_instance = True
+
+
+class HousingProgramSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = HousingProgram
+        include_relationships = True
+        load_instance = True
+
+
+class IntakeQuestionSetSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = IntakeQuestionSet
+        include_relationships = True
+        load_instance = True
+
+
+class ProgramParticipantSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = ProgramParticipant
+        include_relationships = True
+        load_instance = True
+
+
+class ProgramCoordinatorSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = ProgramCoordinator
+        include_relationships = True
+        load_instance = True
+
+
+class GuestGroupSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = GuestGroup
+        include_relationships = True
+        load_instance = True
+
+
+class GuestGroupMemberSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = GuestGroupMember
+        include_relationships = True
+        load_instance = True
+
+
+class HostSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = Host
+        include_relationships = True
+        load_instance = True
+
+
+class HostHouseholdSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = HostHousehold
+        include_relationships = True
+        load_instance = True
+
+
+class HostHouseholdMemberSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = HostHouseholdMember
+        include_relationships = True
+        load_instance = True
+
+
+class CaseStatusSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = CaseStatus
+        include_relationships = True
+        load_instance = True
+
+
+class ProgramCaseSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = ProgramCase
+        include_relationships = True
+        load_instance = True
+
+
+class ProgramCaseStatusLogSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = ProgramCaseStatusLog
+        include_relationships = True
+        load_instance = True
```

Comment:
We don't use any of these schema or models. Can we just keep `HostSchema` and `HousingProgramServiceProviderSchema`? Can we also delete all of the corresponding models in the `database.py` file? Our current database model is actually much simpler than what is written, so leaving these models is just confusing.

Reply:
Are we sure we want to remove models in the database file? They will probably be required at some point.